library/init/meta/simp_tactic.lean

```diff
@@ -58,8 +58,8 @@ lemma and_congr (h₁ : a ↔ c) (h₂ : b ↔ d) : (a ∧ b) ↔ (c ∧ d) := .
 -/
 meta constant simp_lemmas.add_congr : simp_lemmas → name → tactic simp_lemmas
 
-meta def simp_lemmas.append (s : simp_lemmas) (hs : list expr) : tactic simp_lemmas :=
-hs.mfoldl simp_lemmas.add s
+meta def simp_lemmas.append (s : simp_lemmas) (hs : list (expr × bool)) : tactic simp_lemmas :=
```

Comment:
Similar situation here, except we can't even use default argument.  Could we just keep the old API?

Could you maybe also try to build mathlib with this change, just to see what breaks?

Reply:
This function doesn't seem to be called in `mathlib`, and only once in core Lean: `mk_simp_set_core`. I will revert this to the old type (which is all `mk_simp_set_core` needs) and introduce a new version just in case it is picked up in `mathlib`.